static/app/components/banner.tsx

```diff
@@ -32,19 +33,15 @@ class Banner extends React.Component<Props> {
     } = this.props;
 
     return (
-      <BannerWrapper
-        backgroundComponent={backgroundComponent}
-        backgroundImg={backgroundImg}
-        className={className}
-      >
+      <BannerWrapper backgroundImg={backgroundImg} className={className}>
         {backgroundComponent}
         {isDismissable ? (
           <StyledIconClose aria-label={t('Close')} onClick={onCloseClick} />
         ) : null}
         <BannerContent>
           <BannerTitle>{title}</BannerTitle>
           <BannerSubtitle>{subtitle}</BannerSubtitle>
-          <BannerActions>{children}</BannerActions>
```

Comment:
Minor nit why not keep it as `BannerActions` and just change the styled component to be a `ButtonBar`?

Reply:
I replaced it with ButtonBar

Reply:
Yeah but I think BannerActions was still a good name for this component, even if the underlying styled component is now a ButtonBar